smarty/templates/install.tpl

```diff
@@ -43,19 +43,30 @@
 			<fieldset>
 				<legend>LORIS MySQL User</legend>
 			<div>
-				<label for="lorisuser">Username: <input type="text" name="lorismysqluser" placeholder="ie. lorisuser" value="{$lorismysqluser}">
+				<label for="lorismysqluser">Username: <input type="text" name="lorismysqluser" placeholder="ie. lorisuser" value="{$lorismysqluser}">
 			</div>
 			<div>
-				<label for="lorispassword">Password: <input type="password" value="{$lorismysqlpassword}" name="lorismysqlpassword" placeholder="ie. LORISISTHEBEST!!!1">
+				<label for="lorismysqlpassword">Password: <input type="password" value="{$lorismysqlpassword}" name="lorismysqlpassword" placeholder="ie. LORISISTHEBEST!!!1">
+			</div>
+			<div>
+				<label for="lorismysql_already_created">Already Created:
+                <input type="checkbox" name="lorismysql_already_created" value="on" {($lorismysql_already_created == 'on') ? 'checked' : ''}/>
 			</div>
 			</fieldset>
+            <hr/>
+            <div>
+                Check "<strong>Already Created</strong>" only if the appropriate mysql user has already been created!<br/>
+                If you or the web admin wish to give the appropriate privileges to an existing MySQL user, <br/>
+                the privileges are <code>SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, REFERENCES, CREATE TEMPORARY TABLES, LOCK TABLES</code><br/>
+                for the following: <code>`{$dbname}`.*</code>
```

Comment:
I don't think the "If you or the web admin" .. sentence belongs here. It's unlikely to be properly maintained in a template in the installer and anyone who's bypassing the web based installer should already know enough about what they're doing to figure that out from elsewhere.